server/src/main/java/org/elasticsearch/rest/action/cat/RestIndicesAction.java

```diff
@@ -433,9 +435,15 @@ Table buildTable(RestRequest request, IndexMetaData[] indicesMetaData, ClusterHe
             }
 
             table.startRow();
-            table.addCell(state == IndexMetaData.State.OPEN ?
-                (indexHealth == null ? "red*" : indexHealth.getStatus().toString().toLowerCase(Locale.ROOT)) : null);
-            table.addCell(state.toString().toLowerCase(Locale.ROOT));
+
+            String health = null;
+            if (indexHealth != null) {
+                health = indexHealth.getStatus().toString().toLowerCase(Locale.ROOT);
+            } else if (indexStats != null) {
+                health = "red*";
+            }
+            table.addCell(health);
```

Comment:
what if health is null here? How will that render?

Reply:
It renders as an empty cell - there are many other cells like this few lines after this change